wntr/network/model.py

```diff
@@ -311,6 +315,30 @@ def gpvs(self):
         """Iterator over all general purpose valves (GPVs)"""
         return self._link_reg.gpvs
 
+    @property
+    def msx(self):
+        """A multispecies water quality model, if defined"""
+        return self._msx
+    
+    @msx.setter
+    def msx(self, model):
+        if model is None:
+            self._msx = None
+            return
+        from wntr.msx.base import QualityModelBase
+        if not isinstance(model, QualityModelBase):
+            raise TypeError('Expected QualityModelBase (or derived), got {}'.format(type(model)))
+        self._msx = model
+
+    def add_msx_model(self, msx_filename=None):
```

Comment:
Suggest changing keyword arg from msx_filename to msx_file_name to follow WNTR convention (eg wn model constructor uses inp_file_name). @dbhart